diff --git a/fundraiser/modules/fundraiser_upsell/fundraiser_upsell.api.php b/fundraiser/modules/fundraiser_upsell/fundraiser_upsell.api.php
index 8ad4362..76426d6 100644
--- a/fundraiser/modules/fundraiser_upsell/fundraiser_upsell.api.php
+++ b/fundraiser/modules/fundraiser_upsell/fundraiser_upsell.api.php
@@ -99,3 +99,21 @@ function hook_upsell_master_donation_alter(&$master_donation) {
 function hook_fundraiser_upsell_create_recurring_success($donation) {
 
 }
+
+/**
+ * Prevent creation of donation series.
+ *
+ * If any module returns a value, upsell will NOT create a sustainer series.
+ *
+ * @param object $master_donation
+ *   Master donation, the result of the upsell.
+ * @param object $upsold_donation
+ *   Original donation that the upsell was generated from.
+ *
+ * @return string
+ *   Name of module implementing the hook, mainly for identifying which module
+ *   prevents the series creation.
+ */
+function hook_fundraiser_upsell_prevent_series($master_donation, $upsold_donation) {
+  return 'my_module';
+}
diff --git a/fundraiser/modules/fundraiser_upsell/fundraiser_upsell.module b/fundraiser/modules/fundraiser_upsell/fundraiser_upsell.module
index d90c213..79a0f44 100644
--- a/fundraiser/modules/fundraiser_upsell/fundraiser_upsell.module
+++ b/fundraiser/modules/fundraiser_upsell/fundraiser_upsell.module
@@ -656,8 +656,15 @@ function fundraiser_upsell_upsell_ajax_submit($did, $amount) {
   // Allow modules to modify the original upsold donation.
   drupal_alter('upsold_donation', $upsold_donation);
 
-  // Create the recurring donation series.
-  $success = fundraiser_upsell_create_recurring_series($master_donation);
+  // Allow modules to prevent the creation of this series.
+  $prevent_series = module_invoke_all('fundraiser_upsell_prevent_series', $master_donation, $upsold_donation);
+  if ($prevent_series) {
+    $success = TRUE;
+  }
+  else {
+    // Create the recurring donation series.
+    $success = fundraiser_upsell_create_recurring_series($master_donation);
+  }
 
   // Grab and process the thank you content if successful.
   if ($success) {
